app/models/shipit/hook.rb

```diff
@@ -75,6 +75,13 @@ def emit(event, stack, payload)
           stack_id: stack&.id,
           payload: coerce_payload(payload),
         )
+        deliver_internal_hooks(event, stack, payload)
+      end
+
+      def deliver_internal_hooks(event, stack, payload)
+        (Shipit.internal_hook_receivers || []).each do |receiver|
```

Comment:
I think it makes more sense for the internal hook receivers to return `[]` by default rather than checking for nil here.

Reply:
:+1: